### PR TITLE
ENT-4596 properly rotate keys

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
@cf-bottom pr

**Comment by cf-bottom** [comment]
I'm not sure I understand, @Lex-2008.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2663/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2663/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2663/

File: packaging/common/cfengine-hub/preinstall.sh [hunk 1]
```diff
@@ -336,31 +336,25 @@ if [ -d $PREFIX/httpd/php/lib/php/extensions/no-debug-non-zts-20170718 ]; then
   rm $PREFIX/httpd/php/lib/php/extensions/no-debug-non-zts-20170718/*
 fi
 
-# Make a backup of the key CFE_CLIENT_SECRET_KEY, if any, and restore the
-# original file content.
-#
+#sed -i '/$config."CFE_ROBOT_PASSWORD"/s/.*/$config["CFE_ROBOT_PASSWORD"] = "'$CFE_ROBOT_PW'";/' $PREFIX/httpd/htdocs/application/config/cf_robot.php
+
```
[the remark this answers is not in the document]
Comment (Lex-2008) [reply packaging/common/cfengine-hub/preinstall.sh]: Right. It's a temporary comment because this file needs to be dealt separately. Should be gone

**Comment by Lex-2008** [comment]
@cf-bottom please build it again

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2672/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2672/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2672/

**Comment by Lex-2008** [comment]
@cf-bottom please build it again

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2702/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2702/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2702/

**Comment by Lex-2008** [comment]
@cf-bottom please build it again

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2705/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2705/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2705/

**Pull request merged**